domains/merkleroots.go

```diff
@@ -28,3 +28,13 @@ type MerkleRootConfirmation struct {
 	Hash         string                      `json:"hash,omitempty"`
 	Confirmation MerkleRootConfirmationState `json:"confirmation"`
 }
+
+// MerkleRootsResponse is the response object that should be returned from the database when
+// requested from merkleroots api endpoint
+type MerkleRootsResponse struct {
+	MerkleRoot  string `json:"merkleRoot"`
+	BlockHeight int32  `json:"blockHeight"`
```

Comment:
IMHO it would be better to return something like this:
```json
{
  "0": "m3rkL3r00tIdx0",
  "1": "m3rkL3r00tIdx1",
  "2": "m3rkL3r00tIdx2",
}
```

Reply:
actually reverted to the object for better sorting during json marshall from gin